examples/postgres-operator/README.md

```diff
@@ -14,18 +24,22 @@ After looking at several alternatives, Zalando's postgres operator felt like the
 
 1. Clone the Zarf project &mdash; for the example configuration files.
 
-1. Download a Zarf release &mdash; you need a binary _**and**_ an init package, [here](../../docs/workstation.md#just-gimmie-zarf).
+1. Log into [Iron Bank](https://registry1.dso.mil) if you haven't already. (Required for the `minio` images in this example)
```

Comment:
can you remove this, IB images are no longer required for the postgres/minio examples

Reply:
Changing things out from underneath me I see :sweat_smile: